packages/make-styles/src/renderer/createDOMRenderer-node.test.ts

```diff
@@ -1,4 +1,4 @@
-/**
+/*
  * @jest-environment node
  */
```

Comment:
Jest is very picky about its config, and doesn't work if I add a `// eslint-disable` comment above this line. Although it is apparently ok if the config comment is not a jsdoc comment, so this modification makes both jest and lint happy.